HomePage.cs

```diff
@@ -18,28 +18,25 @@ public void startBrowser()
         {
             driver = new ChromeDriver();
             driver.Manage().Window.Maximize();
+            driver.Url = "https://www.saucedemo.com/";
+            System.Threading.Thread.Sleep(2000);
+            LogInPage loginPage = new LogInPage(driver);
+            loginPage.loginValidUser("standard_user", "secret_sauce");
         }
 
         [Test]
         public void TestLogin()
         {
-            driver.Url = "https://www.saucedemo.com/";
-            System.Threading.Thread.Sleep(2000);
             LogInPage loginPage = new LogInPage(driver);
-            loginPage.loginValidUser("standard_user", "secret_sauce");
-
-            /* Perform wait to check the output */
             System.Threading.Thread.Sleep(2000);
             Assert.That("PRODUCTS", Is.EqualTo(loginPage.displayName()));
```

Comment:
If you are using Asset.That() format then your actual value should be in the first place.